test-file.txt

```diff
@@ -1,19 +1,16 @@
 1.
-2.
 3.
 4.
 5.
-6.
+6. Line 2 was deleted, so this is now line 5
+And here's a new line on what is now line 6
 7.
 8.
 9.
 10.
 11.
 12.
-13.
-14.
```

Comment:
This comment on the old line 14 is part of a review which includes a top-level comment.